R/metab.bookkeep.R

```diff
@@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
 		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
 		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
-		freq <- nobs
+		freq <- nobs - 1
```

Comment:
My thinking was that to calculate metabolism for day X, you only use data
from day X.

On Wed, Sep 7, 2016 at 2:52 PM, Luke Winslow <notifications@github.com>
wrote:

> In R/metab.bookkeep.R
> <https://github.com/GLEON/LakeMetabolizer/pull/121#discussion_r77882323>:
>
> > @@ -73,7 +73,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
> >  		warning("datetime not found, inferring sampling frequency from # of observations") # issue a warning (note checks in addNAs)
> >  		# NOTE: because of the checks in addNA's, it is unlikely a user would receive this warning via metab()
> >  		# warning will only be seen through direct use of metab.bookkeep when datettime is not supplied
> > -		freq <- nobs
> > +		freq <- nobs - 1
>
> Why ignore the last deltaDO in the day? Is there a reason to ignore it?
> Perhaps we should really be pushing for midnight to midnight data.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/GLEON/LakeMetabolizer/pull/121/files/c75567a84f7677ee3238146a630d9ff014c790da#r77882323>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFXrfw0zu5Wfeve7RfLkchE7YptT_w6Hks5qnwf2gaJpZM4Jvljp>
> .
>



-- 
Ryan D. Batt
Aquatic Ecologist
Postdoctoral Researcher, Rutgers University
Website: http://limnology.wisc.edu/personnel/batt/